docs/docs/index.md

```diff
@@ -0,0 +1,75 @@
+---
+hide:
+- toc
+---
+![Tracee Logo >](images/tracee.png)
+
+👋 Welcome to Tracee Documentation! To help you get around, please notice the different sections at the top global menu:
+
+- You are currently in the [Getting Started](./) section where you can find general information and help with first steps.
+- In the [Tutorials](./tutorials/overview) section you can find step-by-step guides that help you accomplish specific tasks.
+- In the [Docs](./docs/overview) section you can find the complete reference documentation for all of the different features and settings that Tracee has to offer.
+- In the [Contributing](./contributing/overview) section you can find technical developer documentation and contribution guidelines.
+
+<!-- links that differ between docs and readme -->
+[installation]:./docs/install/
+[docker-guide]:./docker.md
+[kubernetes-guide]:./kubernetes.md
+[prereqs]:./docs/install/prerequisites.md
+<!-- everything below is copied from readme -->
+
+Before moving on, please consider giving us a GitHub star ⭐️. Thank you!
+
+## About Tracee
+
+Tracee is a runtime security and observability tool that helps you understand how your system and applications behave.  
+It is using [eBPF technology](https://ebpf.io/what-is-ebpf/) to tap into your system and expose that information as events that you can consume.  
+Events range from factual system activity events to sophisticated security events detect suspicious behavioral patterns.
+
+To learn more about Tracee, check out the [documentation](https://aquasecurity.github.io/tracee/). 
+
+## Quickstart
+
+To quickly try Tracee use one of the following snippets. For a more complete installation guide, check out the [Installation section][installation].  
+Tracee should run on most common Linux distributions and kernels. For more compatibility information see the [Prerequisites][prereqs] page.
+
+### Using Docker
+
+```shell
+docker run --name tracee \
```

Comment:
If you use this docker command, since it is not in detached mode, it should automatically display the logs -- maybe we could add `-d` in the command or remove the next part of querying the logs `docker logs tracee`?

Reply:
true, so we can just remove the docker logs command IMO